internal/download/artifact.go

```diff
@@ -32,3 +36,47 @@ func Cleanup(directory string) {
 		log.Err(err).Msg("Unable to cleanup previous artifacts")
 	}
 }
+
+// GetDirName returns a target folder for downloading artifacts.
+// It creates a targetDir from suiteName by replacing / \ . and space with hyphen and underscore.
+// If there isn't targetDir under artifacts directory, it returns targetDir. e.g. `chrome_test`
+// If there is an existing targetDir, it finds the maxVersion from the suffix and adds one to that. e.g. `chrome_test.1`
```

Comment:
That's a bit too detailed ;) You don't want to describe every step of the method. That's what the code is for. What you want to describe is the intent/purpose, so that folks know what the method is trying to achieve. The _how_ is in the code.

```suggestion
// GetDirName returns a target folder that's based on a combination of suiteName and the configured artifact download folder.
// The suiteName is sanitized by undergoing character replacements that are safe to be used as a directory name.
// If the determined target directory already exists, a running number is added as a suffix.
```